ltj/settings.py

```diff
@@ -115,11 +115,12 @@ def get_git_revision_hash():
     "ckeditor",
     "rest_framework",
     "rest_framework_gis",
+    "axes",
```

Comment:
Why is `axes` moved to the middle of the installed apps here? I guess it does not break anything, but it seems like an odd placement, not alphabetical, not first of the third-party apps.

Reply:
I usually group the apps into categories as **django apps**, **third-party apps** and **project apps** (like imports). Could be better to order them in alphabetic order within each group, but then there's no automatic tool doing that, so thinking it may not worth the efforts to manually order them. And this project is a bit of an exception because the `django-heluser` apps must be put before all other apps. But I think I could add a few comments here to make it more obvious.